config/experiments.go

```diff
@@ -4,7 +4,6 @@ type Experiments struct {
 	FilestoreEnabled              bool
 	UrlstoreEnabled               bool
 	ShardingEnabled               bool `json:",omitempty"` // deprecated by autosharding: https://github.com/ipfs/kubo/pull/8527
-	GraphsyncEnabled              bool
```

Comment:
I could do like I usually do an make a special erroring type but I don't think it's very requiered, the error on the other wide when it will stop working should be pretty clear (something about protocol ids not matching).

Reply:
For the writable gateway - whose use was a bit unknown - we added a `log.Fatalf` if enabled. This will ensure any current users know about it and act accordingly:

https://github.com/ipfs/kubo/blob/01cc5eab57ed7acab1593280c7b5565d18c8f3ae/cmd/ipfs/daemon.go#L808-L810